docs/README.Windows.md

```diff
@@ -98,8 +98,7 @@ Start the VS2017 installer and click `Individual components`.
 * Under **Development activities** select
   * `Visual Studio C++ core features`
 * Under **SDKs, libraries, and frameworks** select
-  * `Windows 10 SDK (10.0.x.0) for Desktop C++ [x86 and x64]`
-  * `Windows 10 SDK (10.0.x.0) for UWP: C++`
+  * `Windows 10 SDK (10.0.x.0)` (select the latest version)
```

Comment:
```suggestion
  * `Windows 10 SDK (10.0.18362.0) (if compiling for UWP or UWP-ARM)
```

Reply:
that tells me if im compiling for x64 - i dont need any sdk...

when I was trying to compile, I was just doing x64 (not uwp)
 15063 & 16299  didnt work but 17764 did

So maybe we need both?
*  `Windows 10 SDK (10.0.x.0)` (if compiling for x86 and x64)
* `Windows 10 SDK (10.0.18362.0) (if compiling for UWP or UWP-ARM)

Reply:
> that tells me if im compiling for x64 - i dont need any sdk?

No, this tells us that the _default SDK_ is fine. Installing VS always installs a default SDK unless a custom install is used.

The default SDK in VS 2017 is 10.0.17763.0
The default SDK in VS 2019 is 10.0.19041.0

Both are fine to compile x64 desktop.

The instructions only say that additional components need to be installed only in the case of UWP.

Additional components are:

> MSVC v142 - VS 2019 C++ ARM build tools (Latest) (if compiling for UWP-ARM)

> Windows 10 SDK (10.0.18362.0) (if compiling for UWP or UWP-ARM)


Then when compiling UWP and additional components are installed you ends up with two SDK:

**Visual Studio 2017**
SDK 10.0.17763.0 (*) +  SDK 10.0.18362.0

or

**Visual Studio 2019**
SDK  10.0.19041.0 (*)  + SDK  10.0.18362.0

(*) = default

And when compiling x64 more recent SDK installed are auto picked: 
VS 2017 --> 10.0.18362.0  
VS 2019 --> 10.0.19041.0

On the contrary, when compiling UWP | UWP-32 | UWP-ARM the target SDK is always used:
VS 2017 --> 10.0.18362.0 
VS 2019 --> 10.0.18362.0